projects/packages/waf/src/class-waf-runner.php

```diff
@@ -530,8 +584,7 @@ public static function generate_ip_rules() {
 		$allow_list = self::ip_option_to_array( get_option( self::IP_ALLOW_LIST_OPTION_NAME ) );
 		$block_list = self::ip_option_to_array( get_option( self::IP_BLOCK_LIST_OPTION_NAME ) );
 
-		$lists_enabled = (bool) get_option( self::IP_LISTS_ENABLED_OPTION_NAME );
-		if ( false === $lists_enabled ) {
+		if ( false === JETPACK_WAF_IP_LISTS_ENABLED ) {
```

Comment:
This method should always be run within the WordPress context, but I'm defining and using the constant here just for consistency in approach, instead of accessing the option directly.

Reply:
Removed